projects/core/components/hosted-dropdown/test/hosted-dropdown.component.spec.ts

```diff
@@ -126,9 +127,10 @@ describe('TuiHostedDropdown', () => {
                 TuiTextfieldControllerModule,
                 TuiDataListModule,
                 TuiElementModule,
-                TuiRootModule,
+                TuiRootComponent,
             ],
             declarations: [TestComponent],
+            providers: [NG_EVENT_PLUGINS],
```

Comment:
Ideally we would want to have this in every test, since you never know what component relies on plugins. Can we somehow do that?

Reply:
fixed